ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/IPv6/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+##############################
+# Enable IPv6 in sysctl.conf #
+##############################
+
+- sysctl:
+    name: net.ipv6.conf.all.disable_ipv6
+    value: '0'
+    sysctl_set: yes
+    state: present
+    reload: no
+  when:
+    - ansible_distribution == "RedHat"
+  tags: IPv6_sysctl
+
+- sysctl:
+    name: net.ipv6.conf.default.disable_ipv6
+    value: '0'
+    sysctl_set: yes
+    state: present
+    reload: yes
+  when:
+    - ansible_distribution == "RedHat"
+  tags: IPv6_sysctl
```

Comment:
Looks like this is missing a newline character at the end

Reply:
On our machines all CentOS have this set by default while all RHEL needed this PB run to allow IPv6 to function. But yes should work for CentOS the same way so we could add that in the `ansible_distribution` as well as I don't think it would do any harm.

I can add the `adoptopenjdk` tag no problem.

Good eye on the newline will fix that up as well.

Reply:
Looks like the travis check has failed as you've put a full blank line in at the end (The i.e. it ends with two new line characters where you had zero before - it needs one :-)